src/components/Requirements/IncompleteSubReqCourse.vue

```diff
@@ -20,32 +20,30 @@
         @start="onDrag"
         @end="onDrop"
       >
-        <div
-          v-for="(course, index) in coursesWithoutRequirementID"
-          :key="index"
-          class="requirements-courseWrapper"
-        >
-          <course
-            :courseObj="course"
-            :isReqCourse="true"
-            :compact="true"
-            :active="false"
-            class="requirements-course"
-          />
-        </div>
+        <template #item="{ element }">
```

Comment:
Is this a new Vue 3 thing? Does this replace v-for loops somehow?

Reply:
No I think it's only specific to the draggable library